### PR TITLE
Change name

### PR DESCRIPTION
**Pull request by mcsherrylabs**
Update doco for command line version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-software into the hands of Ethereum Classic users, giving them a safe way to create, manage and backup their wallets.    
+The Daedalus release consists of two parts, the integration with the Daedalus Wallet and the release of the command line version
+ following on from the Beta release in August. This branch refers to release candidate 1 of the of the command line version.
  
-This version has been tested on Windows 10 and MacOS 
+The differences are very minor and relate only to how the RPC apis are enabled. In the command line version they are enabled over HTTP
+and in the Daedalus integration version they are forced to use HTTPS and client SSL certificate. The Daedalus version comes with a bundled JVM 
+but the command line version requires a JVM to be installed and available on the target machine.   
+    
+This version has been tested on Windows 10, MacOS and Ubuntu.
```
Comment (LukasGasior1) [inline README.md]: and Fedora :)

File: README.md [hunk 2]
```diff
@@ -19,13 +21,11 @@ This version of the code supports
   - CPU mining 
   - peer discovery 
   - fast sync (download a recent state trie snapshot and all blocks, this is the default behaviour)  
-  - bootstrap sync (download a database for *mantis* preloaded with a recent version of the block chain, **highly recommended** for testing)
-  - regular sync (download and execute every transaction in every block in the chain, this can be very slow and error prone - not recommended) 
+  - bootstrap sync (download a database for *mantis* preloaded with a recent version of the block chain, **highly recommended**)
```
Comment (LukasGasior1) [inline README.md]: block chain -> blockchain?

File: src/universal/conf/network.conf [hunk 3]
```diff
@@ -71,21 +71,21 @@ mantis {
...
 
       # File with the password used for accessing the certificate keystore (used only for https)
       # null value indicates HTTPS is not being used
-      certificate-password-file = "conf/nopassword"
+      # certificate-password-file = null
 
       # Enabled JSON-RPC APIs over the JSON-RPC endpoint
       # Available choices are: eth, web3, net, personal, daedalus
-      # apis = "eth,web3,net,personal,daedalus"
+      apis = "eth,web3,net,personal"
```
Comment (LukasGasior1) [inline src/universal/conf/network.conf]: no `#`?

Comment (mcsherrylabs) [reply src/universal/conf/network.conf]: i disabled daedalus for cli....

File: README.md (hunk 1 above)
Comment (mcsherrylabs) [reply README.md]: (Changed it back to Linux, it's easier!)

File: src/universal/conf/network.conf (hunk 3 above)
Comment (mcsherrylabs) [reply src/universal/conf/network.conf]: @LukasGasior1 @KonradStaniec Any foresee an issue with disabling the daedalus API for the command line ...?

Comment (LukasGasior1) [reply src/universal/conf/network.conf]: nope

**Pull request merged**